mir_perception/mir_object_recognition/ros/src/multimodal_object_recognition_node.cpp

```diff
@@ -94,12 +96,10 @@ MultimodalObjectRecognitionROS::MultimodalObjectRecognitionROS(ros::NodeHandle n
   ROS_WARN_STREAM("[multimodal_object_recognition] target frame: " <<target_frame_id_);
   nh_.param<std::string>("pointcloud_source_frame_id", pointcloud_source_frame_id_, "fixed_camera_link");
 
-  nh_.param<std::string>("logdir", logdir_, "/tmp/");
+  nh_.param<std::string>("logdir", logdir_, "/home/robocup/perception_debug_data/");
```

Comment:
this line should not be changed, since it's a parameter in the launch file anyway